assembl/views/__init__.py

```diff
@@ -371,12 +371,25 @@ def process_export_list(ls):
         "get_description": get_description(request),
         "get_landing_page_image": get_landing_page_image(),
         "private_social_sharing": private_social_sharing(),
-        "get_topic": get_topic(request)
+        "get_topic": get_topic(request),
+        "get_discussion_url": get_discussion_url()
     })
 
     return base
 
 
+def get_discussion_url():
+    from ..auth.util import get_current_discussion
+    discussion = get_current_discussion()
+    if discussion:
+        if (len(discussion.get_discussion_urls()) == 2):
+            return discussion.get_discussion_urls()[1]
+        else:
+            return discussion.get_discussion_urls()[0]
+    else:
+        return "bluenove"
```

Comment:
No, there should be a fallback. If there is no discussion, I think you can give the global URL of the server as absolute fallback. You can get it by using `get_global_base_url` method.